client/editEvents.js

```diff
@@ -257,7 +247,7 @@ Template.editEvents.helpers({
   },
   lastRoundCode: function() {
     var roundId = getLastRoundIdForEvent(this.competitionId, this.eventCode);
-    return Rounds.findOne(roundId).roundCode;
+    return Rounds.findOne(roundId).roundCode();
```

Comment:
There might be some more room for cleanup here. I've created https://github.com/cubing/ccm/issues/244 to remember this.